docs/pages/cv/classification_reference.md

```diff
@@ -0,0 +1,297 @@
+| ------------------------------- | -------------- | ------------------------------------------------------------------- |
+| **Analytical dimension**        | `is_a`         | Defines the metric subtype (what kind of QC metric it *is*)         |
+| **Workflow stage**              | relationship   | Where in the experimental/computational pipeline the metric applies |
+| **Information dependency type** | relationship   | What type of input data the metric needs                            |
+| **Measurement scope**           | relationship   | At what aggregation level the metric summarizes data                |
+| **Acquisition strategy**        | relationship   | Which acquisition/mode or platform it applies to                    |
+| **Quality interpretation type** | relationship   | How to interpret higher/lower/targeted values                       |
+| **Metric value type**           | relationship   | How the values are structurally represented (single, tuple, etc.)   |
+
+**Rule of thumb:**
+Every QC metric has exactly one `is_a` (analytical dimension) and one value from each of the other six relationship dimensions.
+
+## Part 1 — Inheritance: Analytical dimension
+
+**What it is:**
+The analytical dimension defines the *type of QC metric*.
+This is the only dimension expressed via **inheritance** (`is_a`) because it establishes the metric's place in the taxonomy.
+
+**How to use it:**
+Choose exactly one of the following as the metric's `is_a` parent:
+
+#### Subclasses
+
+- **Acquisition coverage metric:** how comprehensively data were collected (e.g., scan counts, sampling density).
+- **Mass accuracy metric:** deviation between observed and theoretical _m_/_z_.
+- **Intensity stability metric:** variation of signal intensity over time.
+- **Chromatographic performance metric:** separation performance (e.g., eak width, symmetry, RT reproducibility).
+- **Ionization quality metric:** properties of the precursor ion population (e.g., charge-state distribution, adduct prevalence).
+- **Ion mobility metric:** IMS resolution, drift-time/CCS accuracy and reproducibility.
+- **Spectral quality metric:** quality of individual spectra (e.g., peak density, S/N, completeness).
+- **Fragmentation efficiency metric:** effectiveness of precursor ion fragmentation to produce interpretable spectra.
+- **Isolation purity metric:** precursor isolation selectivity or co-isolation of interfering species.
+- **Identification confidence metric:** reliability of identifications (e.g., FDR, ID rate).
+- **Quantification precision metric:** reproducibility or variability of quantitative results.
+- **Contamination metric:** unwanted signal from contaminants, carryover, or background.
+- **Instrument operational performance metric:** general indicators of instrument health (e.g., vacuum, detector voltage, temperature).
+- **Missingness/completeness metric:** data absence or completeness across features, runs, or studies.
```

Comment:
this has quite considerable overlap with the `workflow` stage and it's potentially a dimension which is hard to cover completely, since to me it's quite fuzzy. This may lead to people cramming their metric into something which "fit's good enough" instead of thinking of a new category value.

What could be a set of values which is
1) easy to categorize
2) does not need extension in the future
3) adds value compared to the other dimensions

, or in other words: what could this dimension represent, making it distinct from others?
(so dimensions are ideally orthogonal)